src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
+If you haven't already, [sign up for a free New Relic
+account](https://newrelic.com/signup) to jump from your application's stack
+trace errors in New Relic One directly to the line of code responsible in your
+IDE.
 
 ## Create an account [#account]
 
-If you already have the CodeStream extension [installed in your IDE](/docs/codestream/start-here/install-codestream), you can start the sign up process from the CodeStream pane.
+If you already have the CodeStream extension [installed in your
+IDE](/docs/codestream/start-here/install-codestream), you can start the sign up
+process from the CodeStream pane.
 
 ![Create an account](./images/CreateAnAccount6.png)
 
-There are two ways to sign up for CodeStream. You can sign up with a set of CodeStream credentials (such as, email address and password). Alternatively, you can sign up using your GitHub, GitLab, or Bitbucket (cloud versions only) account. 
+You can sign up using your email address and a password, or you can sign up using
+your GitHub, GitLab, or Bitbucket account (cloud versions only, although
+CodeStream does have integrations with the on-prem versions once you've
+signed up). If you're a New Relic customer, you can also sign up with a user api
+key from your New Relic One account.
 
-Signing up via your code host also connects your repositories to CodeStream. If the email address you're using with your code host isn't your work email, you should create CodeStream-specific credentials instead.
+If the email address you're using with your code host isn't your work email, you
+should sign up with your work email address instead.
 
-If you sign up via CodeStream, your next step will be to confirm your email address by entering a code sent to you via email. You can paste the code into any of the boxes rather than typing each number individually.
+If you sign up using your email address, your next step will be to confirm the
+email address by entering a code sent to you via email. You can paste the code
+into any of the boxes rather than typing each number individually.
 
 ![Confirm Email](./images/EmailConfirmation1.png)
 
-## CodeStream organizations [#organizations]
-
-A CodeStream organization is where you and your teammates will discuss code. Similar to a Slack workspace, all of the developers in your company are in the same CodeStream organization. CodeStream's activity feed keeps things relevant for you by showing activity related to the code you have open in your IDE. 
-
-The discussions about code build up a knowledge base that is a company-wide resource, so the only reason to have multiple organizations on CodeStream is if you truly need separation. For example, you might have an organization for your day job and another for an open-source project you work on. Or maybe you're a consultant that's a member of different CodeStream organizations for each of your clients.
-
 ## Create or join an organization [#create-org]
```

Comment:
@zuluecho9 Do you have any thoughts on this section?

Reply:
i think that's pretty clear. I think 'organization' is pretty widely used for this type of concept in other companies. And I don't see much chance of it being confused with the New Relic concept of 'organization' (although may be something to keep an eye on if we also start referencing New Relic organizations in these docs). 

One other idea might be to, when it seems unclear what an 'organization' is, to link to this section. I do that in some places in the accounts/orgs docs when I think there's a chance a customer won't be clear about what the concept is exactly and they may think it's being used in some general way.